{{ cookiecutter.repo_name }}/README.md

```diff
@@ -1,157 +1,55 @@
-# {{ cookiecutter.project_name }}
-
-## Overview
+# `{{ cookiecutter.project_name }}`
 
 {{ cookiecutter.overview }}
 
-## Requirements
+> ℹ️ Where this documentation refers to the **root folder** we mean where this README.md is located.
 
-To run the code in this GitHub repository, please make sure your system meets the following requirements:
+- [Getting started](#getting-started)
+  - [Requirements](#requirements)
+{% if cookiecutter.create_secrets_file == "Yes" %}- [Required secrets and credentials](#required-secrets-and-credentials){% endif -%}
+- [Licence](#licence)
+- [Contributing](#contributing)
+- [Acknowledgements](#acknowledgements)
 
-- Unix-like operating system (macOS, Linux, …);
-- [`direnv`](https://direnv.net/) installed, including shell hooks;
-- [`.envrc`](.envrc) allowed/trusted by `direnv` to use the environment variables - see
-[below](#allowingtrusting-envrc);
-{%- if cookiecutter.create_secrets_file == "Yes" %}
-- If missing, [create a `.secrets` file](#creating-a-secrets-file) to store untracked secrets;{% endif %}
-- Python 3.5 or above; and
-- Python packages installed from the `requirements.txt` file.
+## Getting started
 
-Note there may be some Python IDE-specific requirements around loading environment variables, which are not considered
-here.
+To be added.
 
-### Allowing/trusting `.envrc`
+### Requirements
 
-To allow/trust the [`.envrc`](.envrc) run the `allow` command using `direnv` at the top level of this repository.
+{% if cookiecutter.create_secrets_file == "Yes" %}- A `.secrets` file with the [required secrets and credentials](#required-secrets-and-credentials){% endif -%}
+- [Load environment variables][docs-loading-environment-variables] from `.envrc`
 
-```shell script
-direnv allow
-```
+To be added.
```

Comment:
Is this a placeholder for `direnv allow`? Should possibly include something about how this wouldn't work with a PyCharm set-up (1 point to VSCode. :trollface:)

Reply:
I think this is a placeholder for **any** requirements that the creator needs to add in their newly-created project! Although I take your point about PyCharm and `direnv` - feel free to contribute a solution!!

Reply:
I will when I can get time and motivation. 😭

Reply:
Marking this as **Resolved**.